src/refactor_nrepl/ns/slam/hound/search.clj

```diff
@@ -99,26 +101,28 @@
         [(class-or-ns-name fpr)])
       [])))
 
-(defn scan-paths
-  "Takes one or more classpath strings, scans each classpath entry location, and
-  returns a list of all class file paths found, each relative to its parent
-  directory or jar on the classpath."
-  ([cp]
-     (if cp
-       (let [entries (enumeration-seq
-                      (StringTokenizer. cp File/pathSeparator))
-             locs (mapcat expand-wildcard entries)]
-         (mapcat #(path-class-files % %) locs))
-       ())))
-
-(defn- get-available-classes
-  []
-  (->> (mapcat scan-paths (concat (map #(System/getProperty %) ["sun.boot.class.path"
-                                                                "java.ext.dirs"
-                                                                "java.class.path"])
-                                  (map #(.getName %) (cp/classpath-jarfiles))))
-       (remove clojure-fn-file?)
-       (map symbol)))
+(defn path-entries-seq
+  "Split a string on the 'path separator', i.e. ':'. Used for splitting multiple
+  classpath entries."
+  [path-str]
+  (enumeration-seq
+   (StringTokenizer. path-str File/pathSeparator)))
+
+(defn all-classpath-entries []
+  (into (map #(System/getProperty %) ["sun.boot.class.path"
+                                      "java.ext.dirs"
+                                      "java.class.path"])
+        (map #(.getName %) (cp/classpath-jarfiles))))
```

Comment:
I just copied this logic over, although it seems strange to me that this is really the way to get the full classpath.